datadog_checks_dev/tests/tooling/manifest_validator/test_v2_validator.py

```diff
@@ -49,24 +49,6 @@ def setup_route():
     set_root(current_root)
 
 
-@mock.patch(
```

Comment:
Should we just skip this test instead for now, until V1 is fully removed?

Reply:
It just seems broken cc @mgarabed

Reply:
Not familiar with what errors you are seeing.